source/builders.txt

```diff
@@ -0,0 +1,110 @@
+.. _kotlin-sync-builders:
+
+================
+Use Builders API
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. .. toctree::
+
+..    /fundamentals/builders/aggregates
+..    /fundamentals/builders/filters
+..    /fundamentals/builders/indexes
+..    /fundamentals/builders/projections
+..    /fundamentals/builders/sort
+..    /fundamentals/builders/updates
+
+Overview
+--------
+
+This page describes how to use the builders API and demonstrates the utility provided by
+builder classes.
+
+The {+driver-short+} provides type-safe classes and methods that enable developers to
+efficiently build queries and aggregations.
+
+Why Use Builders?
+-----------------
+
+When using the MongoDB shell or plain Kotlin to construct BSON query documents, you are not
+able to identify errors in your code until runtime.
+
+With builder classes, you can write operators as methods and any syntax errors will be
+caught at compile time.
+
+Example
+-------
+
+This section describes three methods of fetching the ``email`` field values of documents
+in the ``users`` collection that meet the following criteria:
+
+- Users that identify as ``female``
+- Users that are older than ``29``
```

Comment:
S: describe by using fields and values instead of personificaiton
```suggestion
- ``gender`` value is ``female``
- ``age`` value is greater than ``29``
```